extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -424,22 +400,22 @@ public function getData( $master = false ) {
 
 			if ( $canDelete ) {
 				$img = '<img class="remove sprite" alt="" src="' . $wgBlankImgUrl . '" width="16" height="16" />';
-				$buttons[] = $img . '<a href="' . $title->getLocalUrl( 'redirect=no&action=delete' ) . '" class="article-comm-delete">' . wfMsg( 'article-comments-delete' ) . '</a>';
+				$buttons[] = $img . '<a href="' . $title->getLocalUrl( 'redirect=no&action=delete' ) . '" class="article-comm-delete">' . wfMessage( 'article-comments-delete' )->escaped() . '</a>';
 
 				$links['delete'] = $title->getLocalUrl( 'redirect=no&action=delete' );
 			}
 
 			// due to slave lag canEdit() can return false negative - we are hiding it by CSS and force showing by JS
 			if ( $wgUser->isLoggedIn() && $commentingAllowed ) {
-				$display = $this->canEdit() ? 'test=' : ' style="display:none"';
+				$display = $title->userCan( 'edit' ) ? 'test=' : ' style="display:none"';
```

Comment:
Is it possible to convert all calls to ```canEdit``` method to ```userCan```?
I see one more method using it here: https://github.com/WikiaVolDev/app/blob/VOLDEV-138/extensions/wikia/ArticleComments/classes/ArticleComment.class.php#L632